.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,191 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+
+jobs:
+    phpunit-doctrine_dbal:
+        name: 'PHP ${{ matrix.php-version }} Doctrine DBAL ${{ matrix.dependencies }} ${{ matrix.symfony-require }}'
+
+        runs-on: ubuntu-20.04
+
+        services:
+            mysql:
+                image: mysql:5.7
+                env:
+                    MYSQL_ALLOW_EMPTY_PASSWORD: yes
+                    MYSQL_DATABASE: phpcr_odm_tests
+                ports:
+                    - 3306:3306
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony-require }}
+
+        strategy:
+            fail-fast: false
+
+            matrix:
+                php-version:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                dependencies: [highest]
+                symfony-require: ['']
+                include:
+                    - php-version: '7.1'
+                      dependencies: lowest
+                    - php-version: '7.4'
+                      dependencies: highest
+                      symfony-require: 2.3.*
+                    - php-version: '7.4'
+                      dependencies: highest
+                      symfony-require: 3.4.*
+                    - php-version: '7.4'
+                      dependencies: highest
+                      symfony-require: 4.4.*
+                      php-bench: true 
+                    - php-version: '8.0'
+                      dependencies: highest
+                      symfony-require: 5.0.*
+                    - php-version: '8.0'
+                      dependencies: highest
+                      symfony-require: 6.0.*
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Globally install symfony/flex
+              if: matrix.symfony-require != ''
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
+
+            - name: Remove dev dependencies not compatible with Symfony 6
```

Comment:
Should be ok with 1.6.4 release, https://github.com/liip/RMT/commit/f14c83070565cd066d27648c75d6448a03a8b825